app_dart/lib/src/service/luci_build_service.dart

```diff
@@ -411,19 +391,18 @@ class LuciBuildService {
     return true;
   }
 
-  /// Sends a [BuildBucket.scheduleBuild] request using [CheckRunEvent]. It
-  /// returns [true] if it is able to send the scheduleBuildRequest or [false]
-  /// if not.
-  Future<bool> rescheduleUsingCheckRunEvent(CheckRunEvent checkRunEvent) async {
+  /// Sends [ScheduleBuildRequest] for [pullRequest] using [checkRunEvent].
+  /// 
+  /// Returns true if it is able to send the scheduleBuildRequest. Otherwise, false.
+  Future<bool> rescheduleUsingCheckRunEvent(github.PullRequest pullRequest, CheckRunEvent checkRunEvent) async {
     final github.RepositorySlug slug = checkRunEvent.repository!.slug();
```

Comment:
remove?

Reply:
Swapped to rely on the `pullRequest`, and i'm going to leave the shorthand here as it's used several places in this method